### DISCUSSION TITLE
Mail.app under Ventura strange won't shift windows without first manual movement

### DISCUSSION THREAD

**Discussion by jonhall** [rxhanson/RectanglePro-Community#170] [Q&A]
Massive user of the fantastic Applications setup. Strange behaviour under Ventura where Mail.app (and only Mail.app) won't respond to Rectangle Pro commands until manually moved (even by as little as one pixel). Two tabs open in the window, and choosing other tab resets to initial position too.

I can live with it, but wonder if this is known about.

![Screenshot 2022-11-14 at 10 18 14](https://user-images.githubusercontent.com/17359/201635255-fb84d9ee-ec42-4dd0-8167-b900fb146c02.jpg)

**Comment by rxhanson** [comment]
Hmmm. I can't reproduce this. Do normal shortcuts work on Mail for you? Like "left half" via keyboard shortcut? If so, it might help if I had the log for when this happens: hold option and select View Logging... in the menu bar menu.

**Comment by jonhall** [comment]
Hi Ryan,

I’ve upgraded to 13.0.1 and this has disappeared.

If it happens again, I’ll be sure to capture a log.

Best wishes, Jon

**Comment by jonhall** [comment]
Ah, it’s back.

Log starts when mail was opened. 

If you need other stuff, please tell me.

Best wishes, Jon

**Reply by rxhanson** [reply]
Just following up on this. I still cannot reproduce it. Any chance you can send the logs you get?

**Comment by jonhall** [comment]
Hi Ryan,

I got so frustrated with Mail.app (not just its interactions with RectanglePro) that I dumped it for Outlook, which doesn’t have those problems.

RectanglePro now works perfectly again.

Best wishes, Jon



On 2023-01-10, 03:42, Ryan Hanson wrote:


Just following up on this. I still cannot reproduce it. Any chance you can send the logs you get?
